### PR TITLE
JOSS paper minor improvements

### PR DESCRIPTION
**Pull request by marcofavoritobi**
## Proposed changes

Do some minor improvements on the draft of the JOSS paper submission.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -124,6 +124,29 @@ The toolkit is written in Python and it has a modular and object-oriented design
...
+      real_data=real_data,
+      model=model,
+      parameters_bounds=bounds,
+      parameters_precision=precisions,
+      loss_function=loss)
+# calibrate for 30 epochs
+params, losses = cal.calibrate(30)
+```
+-->
+
```
Comment (marcofavoritobi) [inline paper/paper.md:149]: this is the outcome I get when using the usual Markdown code block, with `python` syntax highlighter:


Screenshot from 2022-06-28 21-38-05


In particular, note that the highlighting is quite limited, e.g. import keywords are not highlighted.

**Comment by AldoGl** [comment]
Great work thanks Marco. The only thing missing now would be placing the Table on the side of Figure 3. I tried using the trick you applied for figure 1 and 2 but it does not work in the case of figure + table

File: paper/paper.md (hunk 1 above)
Comment (AldoGl) [reply paper/paper.md]: Yes I had obtained something like this when I tried... let's forget about it then

**Comment by marcofavoritobi** [comment]
note, I just removed the commit that attempted to use the code block.

**Comment by AldoGl** [comment]
squashing and merging this now to move on with the submission

**Pull request merged**